sqalx.go

```diff
@@ -215,7 +215,8 @@ type Option func(*node) error
 // SavePoint option enables PostgreSQL Savepoints for nested transactions.
```

Comment:
This comment should be updated if SQLite supports SavePoints

Reply:
good catch - thank you.